components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3281,11 +3281,25 @@ public void updateGroup(Group oldGroup, Group newGroup) throws CharonException,
         }
     }
 
+    @Override
+    public void patchGroup(String groupId, String currentGroupName, Map<String, List<PatchOperation>> patchOperations)
+            throws NotImplementedException, BadRequestException, CharonException, NotFoundException {
+
+        doPatchGroup(groupId, currentGroupName, patchOperations);
+    }
+
     @Override
     public Group patchGroup(String groupId, String currentGroupName, Map<String, List<PatchOperation>> patchOperations,
                             Map<String, Boolean> requiredAttributes) throws NotImplementedException,
             BadRequestException, CharonException, NotFoundException {
 
+        doPatchGroup(groupId, currentGroupName, patchOperations);
+        return getGroup(groupId, requiredAttributes);
+    }
+
+    private void doPatchGroup(String groupId, String currentGroupName, Map<String, List<PatchOperation>> patchOperations) throws
```

Comment:
Shall we add javadoc comments, here and where necessary.

Reply:
We usually add java doc comments for public methods only. So not necessary to add a comment for doPatchGroup method. patchGroup is a overridden method. So a method comment is not required there as well.